administrator/language/de-DE/com_admin.ini

```
@@ -20,6 +25,7 @@ COM_ADMIN_DISABLED_FUNCTIONS="Deaktivierte Funktionen"
 COM_ADMIN_DISPLAY_ERRORS="Fehler anzeigen"
 COM_ADMIN_DOWNLOAD_SYSTEM_INFORMATION_TEXT="Als Text herunterladen"
 COM_ADMIN_DOWNLOAD_SYSTEM_INFORMATION_JSON="Als JSON herunterladen"
+COM_ADMIN_EXTENSION_AVAILABLE="Available"
```

```suggestion
COM_ADMIN_EXTENSION_AVAILABLE="Verfügbar"
```
Passt das an der Stelle?